utils/arrays.ts

```diff
@@ -2,4 +2,25 @@ import { random } from 'squidly/utils/math';
 
 export function randomElement(array) {
   return array[Math.floor(random() * array.length)];
-}
\ No newline at end of file
+};
+
+export function mostFrequentElement(array) {
```

Comment:
Will also want to make this more generic to handle non-primitives, can discuss offline.

Thanks for the addition though!

Reply:
Discussed offline